### PR TITLE
Added hand drawn tile

### PR DESCRIPTION
**Pull request by saarthakchats**
### When you change [things like this in brackets] please remember to delete brackets after you change it.

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood [see CONTRIBUTING.md](https://github.com/fossasia/labyrinth/blob/master/CONTRIBUTING.md)
- [x] Included a Preview link and screenshot showing after and before the changes.
- [x] Included a description of change below.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link**
  **Here you can add your preview link--**
- [Preview Link](http://rawgit.com/saarthakchats/labyrinth/master/index.html)

### CODE REVIEW

**Comment by arjunpandey2003** [comment]
hey can u give a screenshot of the tile

**Comment by arjunpandey2003** [comment]
wait i saw the tile

**Comment by niccokunzmann** [comment]
@saarthakchats Thanks for adding this. Would you like to resolve the merge conflicts?

**Comment by saarthakchats** [comment]
@niccokunzmann 
Resolved, pls merge

**Comment by saarthakchats** [comment]
@niccokunzmann Pls merge, I have resolved conflicts

**Comment by saarthakchats** [comment]
@niccokunzmann Is it fine now?

**Comment by samyakbvs** [comment]
Hey pls share a screenshot

**Comment by saarthakchats** [comment]
@samyakbvs 
screenshot 33
This is the tile

**Comment by niccokunzmann** [comment]
@saarthakchats Please resolve the conflicts.

**Comment by duttaditya18** [comment]
@saarthakchats Add this tile into the game and resolve merge conflicts.

**Comment by saarthakchats** [comment]
@niccokunzmann 
Pls merge now

File: js/levels.js [hunk 1]
```diff
@@ -8,12 +8,22 @@
 
 // createTestLevel is used in /js/gui.js ignoring the error
 // ignore JSHintBear
+
+
+function createTestLevel()
```
Comment (niccokunzmann) [inline js/levels.js]: So, this function will be overwritten by the function below and deleted during execution.

**Comment by niccokunzmann** [comment]
Thanks! I commented and merge now since I do not know all the plans you have and do not need to know.

**Comment by niccokunzmann** [comment]
@saarthakchats I reverted your pull request. This introduced a syntax error and broke the game. I would ask you to review your pull requests if they are correct. This reduces the work I need to do to keep the game working.

**Pull request merged**